extensions/wikia/CreateNewWiki/tasks/ConfigureWikiFactory.php

```diff
@@ -72,7 +72,19 @@ public function getStaticVariables( $siteName, $imagesURL, $imagesDir, $dbName,
 			'wgEnableSectionEdit' => true,
 			'wgOasisLoadCommonCSS' => true,
 			'wgEnablePortableInfoboxEuropaTheme' => true,
-			'wgWikiDescription' => $description
+			'wgWikiDescription' => $description,
+
+			// Enable Discussions
+			'wgEnableDiscussions' => true,
+			'wgEnableDiscussionsNavigation' => true,
+			'wgArchiveWikiForums' => true,
+			'wgEnableRecirculationDiscussions' => true,
+
+			// Features to enable for new wikis, moved here from wgUniversalCreationVariables
```

Comment:
These are from the value of wgUniversalCreationVariables on wikiaglobal wiki. The variable was referring to a lot of nonexistent variable IDs too, which I got rid of.

The reason behind this change was to eschew separate queries to change variables and do it all at once.

Reply:
I am wondering, if we could abstract the initial configuration to a template and separate it from the codebase, so that we don't have a couple of variables here, another couple of variables there. That's for the future. Do you think it's worth doing as part of some general cleanup?